schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -100,26 +120,15 @@
-                  { "$ref": "http://schema.management.azure.com/schemas/2015-05-04-preview/Microsoft.Network.json#/resourceDefinitions/dnszones"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2015-05-04-preview/Microsoft.Network.json#/resourceDefinitions/dnszones_A"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2015-05-04-preview/Microsoft.Network.json#/resourceDefinitions/dnszones_AAAA"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2015-05-04-preview/Microsoft.Network.json#/resourceDefinitions/dnszones_CNAME"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2015-05-04-preview/Microsoft.Network.json#/resourceDefinitions/dnszones_MX"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2015-05-04-preview/Microsoft.Network.json#/resourceDefinitions/dnszones_NS"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2015-05-04-preview/Microsoft.Network.json#/resourceDefinitions/dnszones_PTR"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2015-05-04-preview/Microsoft.Network.json#/resourceDefinitions/dnszones_SOA"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2015-05-04-preview/Microsoft.Network.json#/resourceDefinitions/dnszones_SRV"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2015-05-04-preview/Microsoft.Network.json#/resourceDefinitions/dnszones_TXT"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_A"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_AAAA"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_CNAME"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_MX"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_NS"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_PTR"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_SOA"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_SRV"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_TXT"},
```

Comment:
During my merge, these lines got added twice somehow. It came up during my CR and so I removed the duplicates. The lines below the same as the lines being removed.